core/src/main/java/org/sonarsource/sonarlint/core/telemetry/TelemetryPayload.java

```diff
@@ -49,15 +49,19 @@
   @SerializedName("install_time")
   private final OffsetDateTime installTime;
 
+  @SerializedName("analyses")
+  private final TelemetryAnalyzerPerformancePayload[] analyses;
+
   TelemetryPayload(long daysSinceInstallation, long daysOfUse, String product, String version, boolean connectedMode,
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Constructor has 8 parameters, which is greater than 7 authorized. ([squid:S00107](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS00107))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarlint.core%3Asonarlint-core-parent&branch=telemetry_perf&issues=AWHYGJQ6wd3JK8wdtRe0&open=AWHYGJQ6wd3JK8wdtRe0)

[ISSUE_KEY]: AWHYGJQ6wd3JK8wdtRe0